src/wal/src/config.rs

```diff
@@ -40,3 +40,43 @@ pub enum StorageConfig {
     Obkv(Box<ObkvStorageConfig>),
     Kafka(Box<KafkaStorageConfig>),
 }
+
+impl StorageConfig {
```

Comment:
Can we extract it rather than placing it in each detailed impl config?

Reply:
fixed